tests/console/test_boolean_question.py

```diff
@@ -0,0 +1,119 @@
+import re
+
+import pytest
+
+from tests.utils import DummyConsole
+
+
+def test_boolean_question_yes():
+    """Test that boolean_question returns True when user selects Yes."""
+    console = DummyConsole("y")
+
+    result = console.boolean_question(
+        description="Confirm?",
+        intro="Are you sure?",
+        default=None,
+    )
+
+    assert console.prompts == ["Confirm? y/n? "]
+    assert result is True
+
+
+def test_boolean_question_no():
+    """Test that boolean_question returns False when user selects No."""
+    console = DummyConsole("n")
+
+    result = console.boolean_question(
+        description="Confirm?",
+        intro="Are you sure?",
+        default=None,
+    )
+
+    assert console.prompts == ["Confirm? y/n? "]
```

Comment:
This is an interesting discrepancy - it's not an error of your making, but since we're in the area, we should ensure that that the "no default" case is consistent in using `[]` around the `y/n`:
```suggestion
    assert console.prompts == ["Confirm? [y/n]? "]
```

Reply:
I've made changes to address the discrepancy in the input_boolean method so that all prompts will have the [] wrapping and changed my test case to follow this